content/blog/2018-04-13-rprofile-noam-ross.md

```diff
@@ -0,0 +1,80 @@
+
+---
+
+KO: What is your name, job title, and how long have you been using R?
+
+NR: I'm Noam Ross, I'm a Senior Research Scientist at [EcoHealth Alliance](https://www.ecohealthalliance.org/), a non-profit that works at the intersection of conservation and health. I've done work in R for about 8 years, which is essentially from the start of graduate school (five years) and three years of work.
+
+KO: Did you love it [R] immediately?
+
+NR: No! And I'm mixed on whether I love it now. Ultimately it's using R in the service of the work that I do that I love. It was very frustrating learning R at first. Even eight years ago, the tooling was much much rougher to work with, and R was the first programming language that I used to any significant extent. It took a long time before I was able to express my ideas through code and accomplish the things I do as opposed to fighting with it all the time.
+
+KO: How did you get started with rOpenSci?
+
+NR: [Carl Boettiger](https://twitter.com/cboettig) [one of the rOpenSci co-founders] was my lab mate and he was in his third or fourth year in grad school when I started. He was my example of how to be a grad student. I didn't realize until later that Carl was, as someone once put it, "someone who had drunk all of the kool-aid" in terms of R and also reproducible research, and thus not a typical example! But he was a very good role model and he provided a lot of guidance for my work, and introduced my into the open-source science ecosystem and community. I guess my first real interaction [with rOpenSci] was when he asked me to review an R package. That's when I started really being involved.
+
+KO: Do you have a favorite rOpenSci project you've worked on?
+
+NR: My favorite thing is the onboarding process and peer-review. Most of what I do with rOpenSci is editorial work for that. I've [written](https://www.numfocus.org/blog/how-ropensci-uses-code-review-to-promote-reproducible-science/) and [spoken about it](https://youtu.be/Dc6qnwbJbTU) in a few places. I think it has worked out really well. My hope for it isn't just that it grows, but that we can spin it out and support more groups adopting the process.
+
+KO: What would that look like?
+
+NR: We [rOpenSci] have a limited scope. There are areas of expertise that we, and our network of reviewers have expertise, so we focus on those to have coherent offering and ensure quality. But we don't cover a lot of other areas, such statistical methods. If someone has a package that implements a new statistical method, we don't review it because our pool of reviewers aren't primarily statisticians. We would love to have a group in a statistical sub-field or other computational sub-fields adopt our process. rOpenSci could support them in setting up a reviewer network and with some of our experience and tooling for managing the process.
```

Comment:
Some sentences in this para are missing words:
- "There are areas of expertise that we, and our network of reviewers have expertise,"  add "in"? or "in which we..."
- "such statistical methods" add "as"

Please check the rest of interview to fix similar things